instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Comment.css';
+
+const Comment = props => {
+    return (
+        <div className="comment-text">
+            <span className="comment">{props.comment.text}</span>{' '}
```

Comment:
The order of this should be reversed, right? Username first, text second? Try putting both in a single `div` and then wrap the title in a `b` tag.